server/routes/api/projects.js

```diff
@@ -5,17 +5,17 @@ const acls = require('../authorisations/projects');
 module.exports = (router, prefix) => {
   const base = `${prefix}/events/:eventId/projects`;
   acls.define(base, prefix);
-  // curl curl -H 'Content-Type: application/json' -X POST --data-binary '{ "name": "MyPoneyProject",  "category": "HTML", "users": [{"firstName": "kid1", "lastName": "kido", "dob": "2008-12-01T20:00.000Z", "gender": "male", "type": "member"}, {"firstName": "orga", "lastName": "le", "dob": "1991-12-01T20:00.000Z", "gender": "male", "email": "a@a.com", "phone": "3538123123123", "country": "IE", "type": "supervisor" }]}' http://localhost:3000/api/v1/events/4583d8ab-f970-4c30-a343-e9ad4995ee4b/projects
   // Create
   router.post(base, passport.authenticate('jwt', { session: false }), acls.isAllowed, handlers.post);
   // Load
   router.get(`${base}/:id`, passport.authenticate('jwt', { session: false }), acls.isAllowed, handlers.get);
   // List
   router.get(base, passport.authenticate('jwt', { session: false }), acls.isAllowed, handlers.getAll);
 
-  // curl -v -H 'Content-Type: application/json' -X PATCH --data-binary '{ "answers": { "social_project": true } }' http://localhost:8080/api/v1/events/8d9432fd-d6d0-4dde-b47c-07f1d465e5bb/projects/fb0972d8-7c6a-4325-b610-fcfde67d9fa4\?token\=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJkYXRhIjoiYzZjNzE0NGMtNDZjZS00ODY3LTk1YmEtMGUxODNiYzUxYjFjIiwiaWF0IjoxNTE1NDMzNjUxLCJleHAiOjE1MTU0NjI0NTF9.I4swMLfWkbSb5EHoxPdbYop1ffFB6CMbezVFiufdg88
   // Update
   router.patch(`${base}/:id`, passport.authenticate('jwt', { session: false }), acls.isAllowed, handlers.patch);
+  // Update status
+  router.patch(`${base}/:id/status`, (req, res, next) => { req.app.locals.public = true; return next(); }, acls.isAllowed, handlers.patchStatus);
```

Comment:
Does the guest role not cover this instead of having to do a weird `public = true` hack?

Reply:
No, because it contains an id which is handled by our own policy